svc-licenses/lcommon/common.go

```diff
@@ -209,3 +209,16 @@ func GenericSave(body []byte, table string, key string) error {
 	}
 	return nil
 }
+
+func GetIDsFromURI(uri string) (string, string, error) {
+	lastChar := uri[len(uri)-1:]
+	if lastChar == "/" {
+		uri = uri[:len(uri)-1]
+	}
+	uriParts := strings.Split(uri, "/")
+	ids := strings.SplitN(uriParts[len(uriParts)-1], ".", 2)
```

Comment:
Do not use "." to split, there might be ID from a server that can have multiple "."